src/network/taraxa_capability.cpp

```diff
@@ -70,29 +115,41 @@ std::pair<bool, blk_hash_t> TaraxaCapability::checkDagBlockValidation(DagBlock c
 }
 
 void TaraxaCapability::onConnect(NodeID const &_nodeID, u256 const &) {
-  LOG(log_nf_) << "Node " << _nodeID << " connected";
-  cnt_received_messages_[_nodeID] = 0;
-  test_sums_[_nodeID] = 0;
+  ba::post(io_service_, [=] {
+    LOG(log_nf_) << "Node " << _nodeID << " connected";
+    cnt_received_messages_[_nodeID] = 0;
+    test_sums_[_nodeID] = 0;
+
+    insertPeer(_nodeID, std::make_shared<TaraxaPeer>(_nodeID));
+    sendStatus(_nodeID, true);
+  });
+}
 
-  insertPeer(_nodeID, std::make_shared<TaraxaPeer>(_nodeID));
-  sendStatus(_nodeID, true);
+void TaraxaCapability::interpretCapabilityPacket(NodeID const &_nodeID, unsigned _id, RLP const &_r) {
+  ba::post(io_service_, [=, bb = _r.data().toBytes()] {
+    RLP r(bb);
```

Comment:
bb ? what does that mean ? Isn't it possible to simply copy _r, e.g. [packet_rlp = _r] ?

Reply:
bb - ByteBuffer. Simply copying the RLP object will be a bug, because RLP is a reference type. I thought it is quite clear what that variable represents. What is your suggestion? `rlp_bytes`? `_r_bytes`?